src/app.tsx

```diff
@@ -49,10 +48,7 @@ export class App extends React.Component<Props, ViewState>{
             
             {this.state.view === 'config-bucket' 
             && 
-            <ConfigureBucket
-                onViewChange={this.handler}
-            
-            />}
+            <div> Configure the bucket</div>}
```

Comment:
Only thing I saw different in a previous main is that it says "Successfully changed state to bucket configuration", but I don't think that's important so looks good to me.